.github/workflows/bump-sec-scanners-config-reusable.yml

```diff
@@ -0,0 +1,176 @@
+# This is a reusbale workflow to bump the 'sec-scanners-config.yaml'
+#
+# It will do so by using the script `hack/scripts/render-sec-scanners-config.sh`, that is not part of the workflow.
```

Comment:
(nit) How about we provide the path of the script as workflow input? But I am also fine with this approach

Reply:
I say, let's keep it this way. I want to pass as little variables as possible and `hack/scripts/render-sec-scanners-config.sh` is a reasonable location for all repos.